docs/mp/oci/01_oci.adoc

```diff
@@ -16,63 +16,76 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-:javadoc-base-url-api: {javadoc-base-url}io.helidon.config/io/helidon/oci
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.integrations.oci.sdk.cdi/io/helidon/integrations/oci/sdk/cdi/
 
 = Oracle Cloud Infrastructure Integration
 :h1Prefix: MP
 :description: Helidon OCI Integration
-:keywords: oci
-:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:keywords: oci, cdi
+:cdi-extension-api-url: https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#spi
```

Comment:
If this is for `helidon-3.x` then substitute `2.0` with `3.0`

Reply:
fixed